src/components/global/Playground/index.tsx

```diff
@@ -591,27 +591,29 @@ export default function Playground({
   }
 
   useEffect(() => {
-    const codeSnippets = {};
+    const codeSnips = {};
     Object.keys(code).forEach((key) => {
       if (typeof code[key] === 'function') {
         /**
          * Instantiates the React component from the MDX content for
          * single-file playground examples.
          */
-        codeSnippets[key] = code[key]({});
+        const DynamicComponent = code[key];
+        codeSnips[key] = <DynamicComponent />;
       } else if (typeof code[key] === 'object') {
         /**
          * Instantiates the list of React components from the MDX content for
          * multi-file playground examples.
          */
         const fileSnippets = {};
         for (const fileName of Object.keys(code[key].files)) {
-          fileSnippets[`${fileName}`] = code[key].files[fileName]({});
+          const DynamicFileComponent = code[key].files[fileName];
+          fileSnippets[`${fileName}`] = <DynamicFileComponent />;
```

Comment:
The prior code would cause the page to crash with the following error: `"Invalid hook call. Hooks can only be called inside of the body of a function component."`.